FabricioOjeda Libreria.cpp

```diff
@@ -0,0 +1,387 @@
+        kits->mostrar();
+    }
+};
+
+#endif
+
+//----------------------------------------------------------- LISTAPEDIDOS.H -----------------------------------------------------------
+
+#ifndef LISTAPEDIDOS_H
+#define LISTAPEDIDOS_H
+
+#include "Pedido.h"
+
+class ListaPedidos {
+private:
+    Pedido** listaPedidos;
+    int capacidad;
+    int cantidad;
+
+public:
+    ListaPedidos(int cap = 50) {
+        capacidad = cap;
+        cantidad = 0;
+        listaPedidos = new Pedido * [capacidad];
+    }
+
+    ~ListaPedidos() {
+        for (int i = 0; i < cantidad; i++) {
+            delete listaPedidos[i];
+        }
+        delete[] listaPedidos;
+    }
+
+    void agregarPedido(Pedido* p) {
+        if (cantidad < capacidad) {
+            listaPedidos[cantidad] = p;
+            cantidad++;
+        }
+    }
+
+    void mostrar() {
+        for (int i = 0; i < cantidad; i++) {
+            listaPedidos[i]->mostrar();
+        }
+    }
+};
+
+#endif
+
+//----------------------------------------------------------- LIBRERIA.H -----------------------------------------------------------
+
+#ifndef LIBRERIA_H
+#define LIBRERIA_H
+
+#include "ListaLibros.h"
+#include "ListaKits.h"
+#include "ListaPedidos.h"
+
+class Libreria {
+private:
+    ListaLibros* listaLibros;
+    ListaKits* listaKits;
+    ListaPedidos* listaPedidos;
+
+public:
+    Libreria() {
+        listaLibros = new ListaLibros();
+        listaKits = new ListaKits();
+        listaPedidos = new ListaPedidos();
+    }
+
+    ~Libreria() {
+        delete listaLibros;
+        delete listaKits;
+        delete listaPedidos;
+    }
+
+    ListaLibros* getLibros() { return listaLibros; }
+    ListaKits* getKits() { return listaKits; }
+    ListaPedidos* getPedidos() { return listaPedidos; }
+};
+
+#endif
+//------------------------------------------------------
+//MAIN
+#include <iostream>
+#include "Libreria.h"
+using namespace std;
+
+int main() {
+    // Crear librería
+    Libreria libreria;
+
+    // ================== AGREGAR LIBROS ==================
+
+    Libro* l1 = new Libro(1, "Libro de Asimov", 10);
+    Libro* l2 = new Libro(2, "El Principito", 5);
+    Libro* l3 = new Libro(3, "Don Quijote", 8);
+    Libro* l4 = new Libro(4, "Sherlock Holmes", 6);
+
+    libreria.getLibros()->agregarLibro(l1);
+    libreria.getLibros()->agregarLibro(l2);
+    libreria.getLibros()->agregarLibro(l3);
+    libreria.getLibros()->agregarLibro(l4);
+
+    cout << "===== LISTA DE LIBROS ORDENADOS =====" << endl;
+    cout << endl;
+    libreria.getLibros()->mostrarOrdenados();
+    cout << endl;
+
+    // ================== CREAR KITS ==================
+
+    Kit* k1 = new Kit("Clasicos", 100, l1, l2);
+    Kit* k2 = new Kit("Literatura", 120, l3, l4);
+
+    libreria.getKits()->agregarKit(k1);
+    libreria.getKits()->agregarKit(k2);
+
+    cout << "===== LISTA DE KITS =====" << endl;
+    cout << endl;
+    libreria.getKits()->mostrar();
```

Comment:
Se debe implementar los metodos dentro de la clase libreria
```suggestion
    libreria->mostrarKits();
```